controllers/util/k8sutil.go

```diff
@@ -203,6 +204,20 @@ func GetStorageClusterInNamespace(ctx context.Context, cl client.Client, namespa
 	return &storageClusterList.Items[0], nil
 }
 
+func GetCephClusterInNamespace(ctx context.Context, cl client.Client, namespace string) (*rookCephv1.CephCluster, error) {
+	cephClusterList := &rookCephv1.CephClusterList{}
+	err := cl.List(ctx, cephClusterList, client.InNamespace(namespace), client.Limit(1))
+	if err != nil {
+		return nil, fmt.Errorf("unable to list cephCluster(s) in namespace %s: %v", namespace, err)
+	}
+
+	if len(cephClusterList.Items) == 0 {
+		return nil, fmt.Errorf("no cephCluster found in namespace %s", namespace)
+	}
+
+	return &cephClusterList.Items[0], nil
+}
```

Comment:
What happens when you have both internal and external Ceph clusters?

Reply:
Since cephCluster deployment is controlled by storageCluster, We will not have more than one cephcluster in one namespace